### PR TITLE
Fix any duplicate labels that were caused before #496

### PR DESCRIPTION
**Pull request by joshelser**
My fix only prevented future duplicates from being created but didn't address any duplicate labels already in the internal.labels table.

### CODE REVIEW

**Comment by joshelser** [comment]
Not sure why the deploy.py is failing. Best guess is that `alter table .. swap with` doesn't work in a real app? Or, I'm just missing something obvious...

**Review by rymurr — APPROVED** [review]

**Comment by joshelser** [comment]
Still not sure why this only caused problems in the setup script (and not on its own). Must be something tricky with the manner in which setup script is called.

I also moved calling this procedure into 004_labels (instead of at the end in 091_migrations) because I think this will prevent the need for us to care later if we still have semi-broken labels.

File: bootstrap/004_labels.sql [hunk 1]
```diff
@@ -371,6 +372,41 @@ BEGIN
...
+
+CREATE OR REPLACE PROCEDURE INTERNAL.REMOVE_DUPLICATE_LABELS()
+    RETURNS STRING
+    LANGUAGE SQL
+    EXECUTE AS OWNER
+AS
+$$
+BEGIN
+    begin transaction;
+        create or replace table internal.labels2 like internal.labels;
```
Comment (rymurr) [inline bootstrap/004_labels.sql:384]: DDL happens outside of a transaction .The transaction block is redundant

File: bootstrap/004_labels.sql [hunk 2]
```diff
@@ -371,6 +372,43 @@ BEGIN
...
 
+
+CREATE OR REPLACE PROCEDURE INTERNAL.REMOVE_DUPLICATE_LABELS()
+    RETURNS STRING
+    LANGUAGE SQL
+    EXECUTE AS OWNER
+AS
+$$
+BEGIN
+    create or replace table internal.labels2 like internal.labels;
```
[the remark this answers is not in the document]
Comment (joshelser) [reply bootstrap/004_labels.sql:383]: The rows are exactly the same (all columns) in the instances that I've seen. I can identify the duplicate rows (group by .. having count > 1), but I can't come up with a way to delete all but one of those rows.

[the remark this answers is not in the document]
Comment (joshelser) [reply bootstrap/004_labels.sql:383]: Talked with the SQL-whisperer, Robert. His first reaction was that we're doing gymnastics _because_ we don't have a unique key on these rows -- just make a unique key and move on. I was hesitant to doing this (expecting it to be more involved than it appears, causing more bugs to fix this bug), but I am growing more accepting to paying down that debt.

Robert did come up with another suggestion which was to duplicate the distinct, existing rows in this table with a new MODIFIED_AT time (`insert into internal.labels select distinct ..., current_timestamp as label_modified_at  from internal labels`), and then delete all rows in the table than have a MODIFIED_AT row less than the row we just made (`delete from internal.labels where label_modified_at < timestampadd(minute, 1, current_timestamp)`). More complicated than a single delete, but relies on a delete to do the work.

[the remark this answers is not in the document]
Comment (rymurr) [reply bootstrap/004_labels.sql:383]: +1 to just adding a unique id. If that is too hard I am happy w option 2

**Pull request merged**